contrib/mkimage-yum.sh

```diff
@@ -30,7 +30,9 @@ if [ -f /etc/dnf/dnf.conf ] && command -v dnf &> /dev/null; then
 	yum_config=/etc/dnf/dnf.conf
 	alias yum=dnf
 fi
-install_groups="Core"
+install_groups=('Core')
```

Comment:
Is there a situation where folks might not want `Core` to be included?  (as-is, this change would currently break folks who are using `-g` and there isn't a way for folks to get the old behavior of replacing `Core` with some other set)

If so, this probably needs to be instead later checking that `install_groups` is empty and adding `Core` after the fact.

Reply:
Correct. To have the same behaviour as current, I will change the position of default `install_groups` and add `Core` only if it's empty.